kitsune/customercare/forms.py

```diff
@@ -0,0 +1,28 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _lazy
+
+from kitsune.customercare.zendesk import ZendeskClient, CATEGORY_CHOICES, OS_CHOICES
+
+
+class ZendeskForm(forms.Form):
+    """Form for submitting a ticket to Zendesk."""
+
+    product = forms.CharField(widget=forms.HiddenInput)
+    category = forms.ChoiceField(
+        label=_lazy("What do you need help with?"), choices=CATEGORY_CHOICES
+    )
+    os = forms.ChoiceField(
+        label=_lazy("What operating system does your device use?"), choices=OS_CHOICES
+    )
+    subject = forms.CharField(label=_lazy("Subject"), required=False)
```

Comment:
shouldn't this be required?

Reply:
Oddly enough, zendesk treats subjects as optional:

> Takes a ticket object that specifies the ticket properties. The only required property is `comment`.

from https://developer.zendesk.com/api-reference/ticketing/tickets/tickets/#create-ticket